network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/TerminalImpl.java

```diff
@@ -171,6 +172,8 @@ private boolean connectNodeBreaker(Resource<VoltageLevelAttributes> voltageLevel
                         if (switchAttributes.getKind() == SwitchKind.BREAKER && switchAttributes.isOpen()) {
                             switchAttributes.setOpen(false);
                             index.updateSwitchResource(switchAttributes.getResource());
+                            closedSwitches.add(switchAttributes.getResource().getId());
+                            index.notifyUpdate(index.getSwitch(switchAttributes.getResource().getId()).get(), "open", true, false);
```

Comment:
Do you notify 2 times (because it is also registered in list and then notified) ?

Reply:
I confirm int the database we have twice modifications

Reply:
Yes, it's an error. Corrected.